src/routing_node.rs

```diff
@@ -392,27 +393,11 @@ impl RoutingNode {
             }
         };
 
-        debug!("Matching expected connection {:?} on connect, our state {:?}.\n",
+        debug!("Matching expected connection {:?} on connect, our state {:?}.",
                connection,
                self.state());
         match self.match_expected_connection(&connection, connection_token) {
-            Some(expected_connection) => {
-                match expected_connection {
-                    ExpectedConnection::Request(_, _, signed_token) => {
-                        match SignedMessage::new_from_token(signed_token) {
-                            Ok(signed_message) => {
-                                debug!("Sending {:?} on {:?}.\n", signed_message, connection);
-                                ignore(self.send_hello(connection, None));
-                                ignore(self.send(signed_message));
-                            }
-                            Err(_) => {}
-                        }
-                    }
-                    ExpectedConnection::Response(_, _, _) => {
-                        ignore(self.send_hello(connection, None));
-                    }
-                }
-            }
+            Some(_) => ignore(self.send_hello(connection, None)),
```

Comment:
tomorrow we should match and print debug statements here if we are swallowing the error I think. Maybe a wee TODO?